srcpkgs/alsaequal/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'alsaequal'
+pkgname=alsaequal
+version=0.7.1
+revision=1
+build_style=gnu-makefile
+make_use_env=1
```

Comment:
Value does not matter, but `yes` is used by convention.

Following fixes cross-compilation, but feel suboptimal used with  make_use_env.
```
pre_build() {
	sed 's/:= gcc/:= ${CC}/' -i Makefile
}
```